apps/mull-ui-e2e/src/integration/US-1.6.spec.ts

```diff
@@ -0,0 +1,106 @@
+import 'cypress-file-upload';
+import * as faker from 'faker';
+import { frameSizes } from './../fixtures/frame-sizes';
+
+/**
+ * Helper function to register
+ * @param email
+ * @param password
+ */
+const registerUser = (email: string, password: string) => {
+  cy.visit('http://localhost:4200/register');
+  cy.get('#name').type('Tyler Uno');
+  cy.get('#email').type(email);
+  cy.get('#password').type(password);
+
+  cy.get('.register-button').click();
+  cy.get('.Toastify__toast.Toastify__toast--success', { timeout: 5000 }).should(
+    'have.css',
+    'background-color',
+    'rgb(39, 176, 154)'
+  );
+};
+/**
+ * Helper function to login
+ * @param username
+ * @param password
+ */
+const loginUser = (username: string, password: string) => {
+  cy.get('#email').type(username);
+  cy.get('#password').type(password);
+  cy.get('.login').click();
+  cy.get('.Toastify__toast.Toastify__toast--success', { timeout: 5000 }).should(
+    'have.css',
+    'background-color',
+    'rgb(39, 176, 154)'
+  );
+};
+/**
+ * Helper function to create event
+ * @param date
+ * @param pillId
+ */
+const createEvent = (date, pillId = 0) => {
+  cy.get('#imageFile').attachFile('../fixtures/trashed-park.jpg');
+
+  cy.get('#startTime').type('11:20');
+
+  cy.get('#endTime').type('15:20');
+
+  cy.get('#eventTitle').type(date.toString());
+
+  cy.get('.-today').click();
+  cy.get('.-today').click();
+
+  cy.get('#description').type('test description');
+
+  cy.get('#location').click();
+  cy.get('#location-input-field').should('be.visible');
+
```

Comment:
The tests were really flaky for me when using Current Location. Each test at some point passed for me, but I never had a full run pass completely.

I looked into why US-1.1 always passes for me, and it was because we put in a location as such:
```
cy.get('#location-input-field').type('845 Rue Sherbrooke');
      cy.get('#location-input-field-option-0', { timeout: 3500 }).should(
        'contain.text',
        '845 Rue Sherbrooke'
      );
```

Adding this ensured this E2E test was no longer flaky for me.

Reply:
I decided not to do the above to not make unnecessary API calls. It isn't flaky for me, all tests consistently pass. 
I added a geolocation stub which might help with your issue. Let me know if it works, if not I'll add the API call.

Reply:
This resolved my issue. Thanks!